src/parser/jobs/dnc/modules/DirtyDancing.tsx

```diff
@@ -354,7 +327,7 @@ export default class DirtyDancing extends Module {
 		}
 	}
 
-	private getNotesIcon(ruleFailed: boolean): any {
+	private getNotesIcon(ruleFailed: boolean): TODO {
```

Comment:
Should be able to drop the return type annotation - pretty sure it'll be a `JSX.Element`